test/functional/test_runner.py

```diff
@@ -407,9 +422,7 @@ def check_script_list(src_dir):
     python_files = set([t for t in os.listdir(script_dir) if t[-3:] == ".py"])
     missed_tests = list(python_files - set(map(lambda x: x.split()[0], ALL_SCRIPTS + NON_SCRIPTS)))
     if len(missed_tests) != 0:
-        print("The following scripts are not being run:" + str(missed_tests))
-        print("Check the test lists in test_runner.py")
-        sys.exit(1)
+        print("%sWARNING!%s The following scripts are not being run: %s. Check the test lists in test_runner.py." % (BOLD[1], BOLD[0], str(missed_tests)))
```

Comment:
Maybe make this warning an error on travis?

```py
if os.getenv('TRAVIS')=='true':
  # On travis this warning is an error to prevent merging incomplete commits into master
  sys.exit(1)
```